infra/aws/terraform/registry.k8s.io/accounts.tf

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_iam_user" "registry-k8s-io-access" {
+  name = "${var.prefix}registry-k8s-io-access"
+  path = "/"
+}
+
+resource "aws_iam_access_key" "registry-k8s-io" {
+  user = aws_iam_user.registry-k8s-io-access.name
+}
```

Comment:
Yeah, so having a chat with @BenTheElder about this... the GCP service accounts (equivalent to IAM User in AWS) are managed through Terraform, but Google service account keys (they equivalent to IAM Access Keys in AWS) are not really managed by Terraform, with [one exception](https://github.com/kubernetes/k8s.io/blob/fae645bedee2bc244f18e70b571c98c5e2479383/infra/gcp/terraform/kubernetes-public/k8s-infra-prow.tf#L42), and the keys are stored in Google Secret Manager).

I think for this PR, let's go ahead and remove this block.